lib/src/ruby/module.cc

```diff
@@ -475,15 +477,21 @@ namespace facter { namespace ruby {
         if (_collection.empty()) {
```

Comment:
Some debug messages would be useful to know if external facts loading was disabled on purpose

Reply:
Added some debug messages in the `module::ruby_load_external` method as the `collection& module::facts()` method could spam the debug log.